cms/models/query.py

```diff
@@ -4,14 +4,23 @@
 from cms.exceptions import NoHomeFound
 
 
-class PageQuerySet(QuerySet):
+class PageQuerySet(MP_NodeQuerySet):
```

Comment:
Can we add `select_related` to this queryset calling `super().select_related` but issuing a warning if `node` or `node__...` is selected (and, of course, removing node from the selection string)?

Similar behaviour for `filter`or `order_by` would be great, e.g.:
```
def order_by(self, *fieldnames):
    modified = [field[6:] if isinstance(field, str) and field.startswith("node__") else field for field in fieldnames]
    if modified != fieldnames:
        warnings.warn(RemovedInDjangoCMS43, "As of django CMS 4.2 the Page model does not have a node property any more. See (link to docs) for more information. This will cause an error in django CMS 4.3 and above.", stacklevel=2)
    return super().order_by(*modified)
```